documentation/en/api-v1-unstable-methods.md

```diff
@@ -8626,6 +8685,61 @@ Response:
 }
 ```
 
+## Subscribe
+
+
+### SubscribeActorEvents
```

Comment:
How does this JSON RPC endpoint differ from `GetActorEvents` described above? They both seem to provide the same functionality when called over JSON RPC: "Give me a list of events you know about now". I would expect `SubscribeActorEvents` to provide a long-living stream where new events are returned as they are added to the chain.

I am not familiar with ETH JSON RPC, maybe I am missing something?

Reply:
@bajtos 

1. The `GetActorEvents ` is a request/response API. You send it an event filter for the events you are interested in and you get back a response containing all the events that match your filter. 

2. The `SubscribeActorEvents ` is a streaming/subscription API. You send it an event filter and it sends back a long lived stream of events. Events are written to this stream as they are emitted/reverted. You can also use this API to get back a list of "historical events that match the filter" before it starts sending real time events.

Let me know if you need more information.

Reply:
> @bajtos
> 
Thank you for the explanation, @aarshkshah1992! What you describes matches my mental model of what I would expect from these APIs.

Could you perhaps capture the information from your comment in the docs?

My confusion comes from the fact that the example response for `SubscribeActorEvents` looks exactly the same as the example response for `GetActorEvents`.

As a user of this new API, I would find helpful if the docs for `SubscribeActorEvents` provided an example of "long lived stream of events". (If this is a well-known concept, then a link to existing docs describing it is sufficient.)

I am approaching this as a newbie user with little knowledge about ETH JSON RPC API and how Filecoin/Lotus works under the hood. Maybe your work is for a different audience. In which case, feel free to ignore my suggestions.

Reply:
@bajtos Yes, that's a good point. Will ping you once we solid docs available for these new APIs.